backend/backend/properties.py

```diff
@@ -254,3 +269,34 @@ def _is_in_monopoly(self):
         if result[0] == 1:
             is_monopoly = True
         return is_monopoly
+
+    def rails_owned(self):
+        """
+        Returns:
+            int: how many railroads are owned by the owner of self.
+        """
+        with self._conn.cursor() as cursor:
+            cursor.execute('SELECT COUNT(*) FROM `properties` ',
+                           'WHERE `game_id` = %s ',
+                           'AND `property_type` = `railroad` ',
+                           'AND `player_id` = %s',
+                           (self._gid, self._owner))
+            result = cursor.fetchone()
+            return result[0] - 1
```

Comment:
Why is there a `- 1` after the result here? As far as I know, the count will get the exact number of rows from the database

Reply:
The only use I had for it was in calculating the rent, 25 * 2**(rails owned -1). Moving the -1 to there now.